GitLab/CacheKeyUnion/Type.dhall

```diff
@@ -0,0 +1 @@
+let CacheKey = ../CacheKey/Type.dhall in < CacheKey : CacheKey | Text : Text >
```

Comment:
Rename the CacheKey in this union to Files, like so:

`< Files: CacheKeyFiles | Text : Text >`